### DISCUSSION TITLE
How to read a property from a imported shapefile and apply a color range based on values?

### DISCUSSION THREAD

**Discussion by federicomarcantognini** [domlysz/BlenderGIS#772] [Q&A]
Hi all,
I have a shapefile imported as separated geometries each with custom attributes (or properties in Blender), now 2 linked questions:

1. How to read a property from an imported shapefile in the node editor? And,
2. How can I apply a colour range based on those values? 

Let's say I need access to the propriety 'TOT11' within the node editor as shown in the image below:
![image](https://user-images.githubusercontent.com/19288193/228201297-ecaa373d-c812-4c6d-adb0-81f46ae9afe1.png)

Many thanks!

**Comment by ccsuehara** [comment]
@federicomarcantognini hi! Were you able to figure this out? Thanks
